shotover-proxy/src/frame/cassandra.rs

```diff
@@ -111,12 +111,9 @@ impl CassandraFrame {
                         ResResultBody::SetKeyspace(set_keyspace) => CassandraOperation::Result(
                             CassandraResult::SetKeyspace(Box::new(set_keyspace)),
                         ),
-                        ResResultBody::Prepared(prepared) => {
-                            tracing::error!("prepared: {:#?}", prepared);
-                            CassandraOperation::Result(CassandraResult::Prepared(Box::new(
-                                prepared,
-                            )))
-                        }
+                        ResResultBody::Prepared(prepared) => CassandraOperation::Result(
+                            CassandraResult::Prepared(Box::new(prepared)),
```

Comment:
Because BodyResResultPrepared is a large struct.
If we remove the box then clippy complains:
![image](https://user-images.githubusercontent.com/5120858/153811125-65e969e0-2566-4dbc-a2a8-6087abfe43b8.png)

Reply:
Oh right, maybe add a comment like we did for `ASTHolder`. In case someone comes along and tries to fix it later.